.github/workflows/build-envoy-images-release.yaml

```diff
@@ -125,7 +128,7 @@ jobs:
       - name: Prep for build
         run: |
           echo "${{ github.sha }}" >SOURCE_VERSION
-          echo "ENVOY_VERSION=$(cat ENVOY_VERSION)" >> $GITHUB_ENV
+          echo "ENVOY_MINOR_RELEASE=$(cat ENVOY_VERSION | sed 's/envoy-\([0-9]\+\.[0-9]\+\)\..*/v\1/')" >> $GITHUB_ENV
```

Comment:
`ENVOY_MINOR_RELEASE` is already set as `v1.23` above, no need to set it here? Or should we make the workflow fail if `ENVOY_VERSION` file does not agree with env var `ENVOY_MINOR_RELEASE`?

Reply:
ENVOY_VERSION is removed as per one of above convo. However, I have added minor release in binary version check step, so it should catch if there is any mismatch.